### ISSUE TITLE
Generated reports not compatible with allure2

### ISSUE THREAD

**Issue by cinderblock** [zaqqaz/jest-allure#46]
I'm trying to generate allure reports from the results but allure is giving me an error:  
`Unknown root element ns2:test-suite`.

[Example results](https://github.com/cinderblock/github-action-working-directory/tree/allure-results/refs/heads/master):
```xml
<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1580811412000' stop='1580811412079'>
    <name>GitHub Actions Test</name>
    <title>GitHub Actions Test</title>
    <test-cases>
        <test-case start='1580811412001' status='passed' stop='1580811412003'>
            <name>action.yml exists</name>
            <title>action.yml exists</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
    </test-cases>
</ns2:test-suite>
```

Looking around, it looks like maybe allure2 uses `json` test results instead of `xml`. Maybe this plugin is only for allure1? But the readme references "allure2". I feel like I'm missing something obvious but I see no references to allure version problems in these issues.

Have people been successful using this with allure2?

My project is FOSS. Checkout the allure verbose [output here](https://github.com/cinderblock/github-action-working-directory/runs/425074309). Expand the "Generate Allure Report" step to see the output.

**Comment by zaqqaz** [comment]
Nope, this plugin definetely supports allure 2 https://github.com/allure-framework/allure2 but I never tried this with the first version of the reporter.

**Comment by cinderblock** [comment]
I linked to the repo that is exhibiting the problem with full build logs. I'd be happy to point you to specific things you're looking for.

I believe I'm on allure 2:

https://github.com/cinderblock/github-action-working-directory/blob/ea36ff7ed5392edbdc3a51a86fc71055220f092d/package-lock.json#L1268-L1273

----

I ended up fixing this by creating my own allure 2 jest result reporter following the allure docs:

https://github.com/cinderblock/github-action-working-directory/blob/master/jest.allure2.js#L1-L10

I am not actually using `jest-allure` anymore as this was simple enough for me.

**Comment by zaqqaz** [comment]
@cinderblock got it.

If you still need some additional features of jest-allure you can try clone & run tests https://github.com/zaqqaz/visual-unit-tests here and then type allure serve, I believe that everything should be ok, if not please let me know.
Thank you!

**Comment by ryparker** [comment]
Are there any plans on upgrading to the most recent allure version (currrently: 2.0.0-beta.6)? The XML allure results have some drawbacks and do not support pdf attachments.

**Comment by ryparker** [comment]
I ended up forking this repository and implementing the upgraded support. https://github.com/ryparker/jest-allure2
